### PR TITLE
Implementing write access to metric database via REST API

### PR DESCRIPTION
**Pull request by oxisto**
This adds two new RPC calls to create and update metrics via gRPC. They are also exposed via the REST API

### CODE REVIEW

File: api/assessment/metric.go [hunk 1]
```diff
@@ -2,10 +2,17 @@ package assessment
 
 import (
 	"encoding/json"
+	"errors"
+)
+
+var (
+	//ErrMetricIdMissing   = errors.New("metric id is missing")
```
Comment (garuppel) [inline api/assessment/metric.go:9]:
```suggestion
```

File: service/orchestrator/metrics.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	err = json.Unmarshal(b, &metrics)
+	if err != nil {
+		return fmt.Errorf("error in JSON marshal: %w", err)
+	}
+
+	return nil
+}
+
+// CreateMetric creates a new metric in the database.
+func (*Service) CreateMetric(_ context.Context, req *orchestrator.CreateMetricRequest) (metric *assessment.Metric, err error) {
```
Comment (lebogg) [inline service/orchestrator/metrics.go:59]: StoreMetric would be more consistent with other RPCs like StoreAssessmentResult or StoreEvidence

Comment (oxisto) [reply service/orchestrator/metrics.go:59]: I would argue, that this creates a new metric and store assessment result stores an already existing evidence. It is also more consistent with the style guide for CRUD operations.

Comment (lebogg) [reply service/orchestrator/metrics.go:59]: Ok, sold!

**Review by lebogg — APPROVED** [review]

**Pull request merged**